src/Catalyst.Core.Modules.Web3/Controllers/LedgerController.cs

```diff
@@ -39,37 +42,46 @@ public sealed class LedgerController : Controller
     {
         private readonly IDeltaHashProvider _deltaHashProvider;
         private readonly IDfs _dfs;
+        private readonly IMapperProvider _mapperProvider;
         private readonly ILogger _logger;
 
-        public LedgerController(IDeltaHashProvider deltaHashProvider, IDfs dfs, ILogger logger)
+        public LedgerController(IDeltaHashProvider deltaHashProvider, IDfs dfs, IMapperProvider mapperProvider, ILogger logger)
         {
             _deltaHashProvider = deltaHashProvider;
             _dfs = dfs;
+            _mapperProvider = mapperProvider;
             _logger = logger;
         }
 
         [HttpGet]
         public async Task<JsonResult> GetLatestDelta(DateTime? asOf)
         {
             var latest = _deltaHashProvider.GetLatestDeltaHash(asOf?.ToUniversalTime());
-            byte[] dfsContent = null;
             try
             {
                 using (var fullContentStream = await _dfs.ReadAsync(latest))
                 {
-                    dfsContent = await fullContentStream.ReadAllBytesAsync(CancellationToken.None);
+                    var contentBytes = await fullContentStream.ReadAllBytesAsync(CancellationToken.None);
```

Comment:
conifgureawait = false?
I feel like I am Codacy 😂

Reply:
yep all async operations need to .configureawait(false)

Reply:
done